Misc/NEWS.d/next/Library/2018-07-02-05-59-11.bpo-34019.ZXJIife.rst

```diff
@@ -0,0 +1,2 @@
+Correct the arguments passed to Opera Browser when opening a new URL
```

Comment:
nitpick: Would you mind to start the entry with "webbrowser: ..."? It's just easier to quickly identify the context of the change, since we have long changelogs between two Python major versions (like 3.6 .. 3.7).

Reply:
Not a problem :) I added it.